src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -288,4 +293,24 @@ private function hardCopy($originDir, $targetDir)
 
         return self::METHOD_COPY;
     }
+
+    private function getPublicDirectory(ContainerInterface $container)
+    {
+        $kernelProjectDir = $container->getParameter('kernel.project_dir');
+        $composerFilePath = $kernelProjectDir.'/composer.json';
+
+        if (file_exists($composerFilePath)) {
```

Comment:
I'd suggest to rewrite this nested "ifs" block using early return blocks (`if (!file_exists(...)) {return; }` etc.)

Reply:
done. also did use now replaceArgument instead of adding a unneeded parameter to the container.